draft-ietf-quic-multipath.md

```diff
@@ -1386,68 +1386,66 @@ TBD-13 (experiments use 0x004e4f5f4349445f) | NO_CID_AVAILABLE | Path abandoned
 on source address spoofing as outlined in {{Section 21.5.4 of QUIC-TRANSPORT}} equally apply.
 Similarly, the anti-amplification limits as specified in {{Section 8 of QUIC-TRANSPORT}} need to be
 followed to limit the amplification risk.
 
-However, while {{QUIC-TRANSPORT}} only allows the use of one path simultaneously
-and therefore only one path migration at the time should be validated,
-this extension allows for multiple open paths, that could in theory be migrated
-all at the same time, and it allows for multiple paths that could be initialized
-simultaneously. Therefore, each path could be used to further amplify an attack.
-Endpoints need to limit the number of maximum paths and might consider
+{{QUIC-TRANSPORT}} only allows the use of one path
+and the number of concurrent path validation attempts is
+limited by number of issued connection IDs.
+This extension, however, allows for multiple open paths that could in theory be migrated
+all at the same time. Further multiple paths could be initialized
+simultaneously. Each open path could be used to further amplify an attack.
+Therefore, endpoints need to limit the maximum number of paths and might consider
 additional measures to limit the number of concurrent path validation processes
 e.g. by pacing them out or limiting the number of path initiation attempts
 over a certain time period.
 
 
-## Use of Transport Layer Security and the AEAD Encryption Nonce
+## Cryptographic handshake and AEAD Nonce
 
 The multipath extension as specified in this document is only enabled after a
 successful handshake when both endpoints indicate support for this extension.
 Respectively, all new frames defined in this extension are only used in 1-RTT packets.
 As the handshake is not changed by this extension, the transport security mechanisms
 as specified in {{QUIC-TLS}}, such as encryption key exchange and peer authentication,
-remain unchanged as well and the respective security considerations in {{QUIC-TLS}} applied unaltered.
-Note that with the use of this extension, multiple nonces can be in use simultaneously
-for the same AEAD key.
-
-Further note, that the limits as discussed on Appendix B of {{QUIC-TLS}}
+remain unchanged. As such, the respective security considerations in {{QUIC-TLS}} apply unaltered. However, note that the limits as discussed on Appendix B of {{QUIC-TLS}}
 apply to the total number of packets sent on all paths.
 
 This specification changes the AEAD calculation by using the path identifier as part of
-AEAD encryption nonce (see {{nonce}}). To ensure a unique nonce, path identifiers
-are limited to 32 bits and cannot be reused for another path in the same connection.
-
+AEAD nonce (see {{nonce}}). To ensure a unique nonce, path identifiers
+are limited to 32 bits and cannot be reused for another path of the same connection.
+However, note that with the use of this extension, multiple nonces can be in use simultaneously
+for the same AEAD key.
```

Comment:
This proposed change removes the last sentences but also edits the previous sentence a bit.
```suggestion
AEAD nonce (see {{nonce}}). To ensure unique nonces for simultaneous use over all paths, path identifiers
are limited to 32 bits and cannot be reused for another path of the same connection.
```
Does this work? @huitema @qdeconinck ?

Reply:
I don't think that helps. Just remove the last sentence, from "However...".

Reply:
But yes, do replace "a unique nonce" by "unique nonces".

Reply:
```suggestion
AEAD nonce (see {{nonce}}). To ensure unique nonces, path identifiers
are limited to 32 bits and cannot be reused for another path of the same connection.
```